deployment/docker-compose.yml

```diff
@@ -205,7 +205,7 @@ services:
       - /root/.ssh:/root/.ssh:ro
     labels:
       - "traefik.enable=true"
-      - "traefik.frontend.rule=Host: services-dev.raiden.network"
+      - "traefik.frontend.rule=Host: services-test.raiden.network"
```

Comment:
Do we need to overwrite this in `deployment/docker-compose.latest.yml` as well?

Reply:
Good point! I didn't really care much for the build so far, so I missed it. Fixed now.